src/components/Inputs/DepositAmountInput/DepositAmountInput.tsx

```diff
@@ -15,6 +15,8 @@ interface IProps {
   blockerInfo?: string
   decimalsLimit: number
   onBlur?: () => void
+  delta: number
```

Comment:
i think clearer name for this prop would be better

Reply:
updated